### ISSUE TITLE
Add transparency sliders on the layers

### ISSUE THREAD

**Issue by Gnafu** [slms4redd/slms-geo-portal#27]
Add the option for the users to adjust each layer's transparency

**Comment by sgiaccio** [comment]
What about putting a slider in the same div as the legend so that it's hidden by default?

**Comment by Gnafu** [comment]
Good idea, we will need to accommodate for layers that have no legend.
